xml/cap_depl_install_caasp.xml

```diff
@@ -410,12 +410,18 @@ pvc/test-sc-persistent   Bound     pvc-c464ed6a-3852-11e8-bd10-90b8d0c59f1c   1G
    volume claims when you're finished:
   </para>
 
-<screen>&prompt.user;kubectl delete pv/pvc-c464ed6a-3852-11e8-bd10-90b8d0c59f1c
-persistentvolume "pvc-c464ed6a-3852-11e8-bd10-90b8d0c59f1c" deleted
-
-&prompt.user;kubectl delete pvc/test-sc-persistent
+<screen>&prompt.user;kubectl delete pvc/test-sc-persistent
 persistentvolumeclaim "test-sc-persistent" deleted</screen>
 
+  <tip>
+    <title>Deleting persistent volume</title>
```

Comment:
Needs to be updated to title-style capitalization

https://doc.opensuse.org/products/opensuse/Styleguide/opensuse_documentation_styleguide_sd/#sec.capitalization